src/main/java/org/assertj/core/api/LongAdderAssert.java

```diff
@@ -0,0 +1,462 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2020 the original author or authors.
+ */
+package org.assertj.core.api;
+
+import org.assertj.core.data.Offset;
+import org.assertj.core.data.Percentage;
+import org.assertj.core.internal.ComparatorBasedComparisonStrategy;
+import org.assertj.core.internal.Longs;
+import org.assertj.core.util.CheckReturnValue;
+import org.assertj.core.util.VisibleForTesting;
+
+import java.util.Comparator;
+import java.util.concurrent.atomic.LongAdder;
+
+import static org.assertj.core.error.ShouldHaveValue.shouldHaveValue;
+import static org.assertj.core.error.ShouldNotContainValue.shouldNotContainValue;
+
+/**
+ * Assertions for {@link java.util.concurrent.atomic.LongAdder}.
+ *
+ * @author Grzegorz Piwowarek
+ */
+public class LongAdderAssert extends AbstractAssert<LongAdderAssert, LongAdder> {
```

Comment:
Being `LongAdder` a `java.lang.Number`, should `LongAdderAssert` implement `NumberAssert`?

Reply:
Good idea, I made it happen: https://github.com/joel-costigliola/assertj-core/pull/1812/commits/74c66ab398be8cf2d347651b01f8758216d75abd